Jenkinsfile

```diff
@@ -278,8 +278,7 @@ pipeline {
                         stage("Build dev image") {
                             steps {
                                 sh '''
-                                make bundles/buildx
-                                bundles/buildx build --load --force-rm --build-arg APT_MIRROR -t docker:${GIT_COMMIT} .
+                                docker buildx build --load --force-rm --build-arg APT_MIRROR -t docker:${GIT_COMMIT} .
```

Comment:
We should clean up the `--force-rm` flags (in a follow-up). Probably can remove the `APT_MIRROR` as well (don't think we set it anywhere in our CI, but would have to verify that one).